build/ical.js

```diff
@@ -6518,7 +6518,17 @@ ICAL.Event = (function() {
     },
 
     get endDate() {
-      return this._firstProp('dtend');
+      var endDate = this._firstProp('dtend');
+      if (!endDate) {
+          var duration = this._firstProp('duration');
+          endDate = this.startDate.clone();
+          if (duration) {
+              endDate.addDuration(duration);
+          } else if (endDate.isDate) {
+              endDate.addDuration(new ICAL.Duration({ days: 1 }));
```

Comment:
Instead of constructing a new `ICAL.Duration` just for the purpose of incrementing the day, could you just increment using the `day` property?